### PR TITLE
Fix logging setup for e2e tests

### LINKED ISSUE #460: Fix the logging in e2e tests
### Description

Currently the logs are not coming in console while running e2e tests

### Importance

must-have

### Location

pkg/test/e2e

### Suggestions for an improvement

Fix the logging infrastructure

### PR DESCRIPTION
**Pull request by AKamyshnikova**
Signed-off-by: akamyshnikova <akamyshnikova@mirantis.com>

### Change log description
Update zookeepercluster_util.go to use logger instead of t.Log
Setup logger in BeforeSuite
Add  "-args -ginkgo.v" to test execution command for proper logging output
 Use rbac.authorization.k8s.io/v1
Drop unused test/e2e/resources/rbac-operator.yaml

### Purpose of the change
Fixes #460 
Fix logging during e2e execution and some other cleanup 

### How to verify it
Run e2e tests

### CODE REVIEW

**Comment by AKamyshnikova** [comment]
@anishakj Check this approach, please

**Comment by anishakj** [comment]
@AKamyshnikova Changes are looking good. Could you also  please update `rbac.authorization.k8s.io/v1beta1` to  `rbac.authorization.k8s.io/v1` as all the other resources are updated

File: pkg/test/e2e/e2eutil/zookeepercluster_util.go [hunk 1]
```diff
@@ -50,55 +49,53 @@ func CreateCluster(t *testing.T, k8client client.Client, z *api.ZookeeperCluster
...
+	logger.Info(fmt.Sprintf("waiting for cluster pods to become ready: %s", z.Name))
 	err := wait.Poll(RetryInterval, ReadyTimeout, func() (done bool, err error) {
-		cluster, err := GetCluster(t, k8client, z)
+		cluster, err := GetCluster(logger, k8client, z)
 		if err != nil {
 			return false, err
 		}
 
-		t.Logf("\twaiting for pods to become ready (%d/%d), pods (%v)", cluster.Status.ReadyReplicas, size, cluster.Status.Members.Ready)
+		logger.Info(fmt.Sprintf("\twaiting for pods to become ready (%d/%d), pods (%v)", cluster.Status.ReadyReplicas, size, cluster.Status.Members.Ready))
```
Comment (anishakj) [inline pkg/test/e2e/e2eutil/zookeepercluster_util.go:98]: could you please remove the `\t` from here

Comment (AKamyshnikova) [reply pkg/test/e2e/e2eutil/zookeepercluster_util.go:98]: Done

File: pkg/test/e2e/e2eutil/zookeepercluster_util.go [hunk 2]
```diff
@@ -110,25 +107,24 @@ func WaitForClusterToBecomeReady(t *testing.T, k8client client.Client, z *api.Zo
...
+		cluster, err := GetCluster(logger, k8client, z)
 		if err != nil {
 			return false, err
 		}
 
 		_, upgradeCondition := cluster.Status.GetClusterCondition(api.ClusterConditionUpgrading)
 		_, errorCondition := cluster.Status.GetClusterCondition(api.ClusterConditionError)
 
-		t.Logf("\twaiting for cluster to upgrade (upgrading: %s; error: %s)", upgradeCondition.Status, errorCondition.Status)
+		logger.Info(fmt.Sprintf("\twaiting for cluster to upgrade (upgrading: %s; error: %s)", upgradeCondition.Status, errorCondition.Status))
```
Comment (AKamyshnikova) [inline pkg/test/e2e/e2eutil/zookeepercluster_util.go:127]: I assume here should be dropped as well

**Comment by AKamyshnikova** [comment]
Added commit with rbac switch

File: test/e2e/resources/rbac-operator.yaml [hunk 3]
```diff
@@ -48,7 +48,7 @@ rules:
 ---
 
 kind: ClusterRoleBinding
-apiVersion: rbac.authorization.k8s.io/v1beta1
```
Comment (anishakj) [inline test/e2e/resources/rbac-operator.yaml:51]: can we remove `creationTimestamp: null`  from deployment section

File: test/e2e/resources/rbac-operator.yaml [hunk 4]
```diff
@@ -6,7 +6,7 @@ metadata:
 ---
 
 kind: ClusterRole
-apiVersion: rbac.authorization.k8s.io/v1beta1
+apiVersion: rbac.authorization.k8s.io/v1
 metadata:
```
Comment (anishakj) [inline test/e2e/resources/rbac-operator.yaml:10]: This file is  not used any where, as we are not using `operator-sdk` for tests.

Comment (AKamyshnikova) [reply test/e2e/resources/rbac-operator.yaml:10]: Yes, I missed dropping it in main commit, will do it in this one, I guess.

**Comment by anishakj** [comment]
@AKamyshnikova Please create an issue and link in this PR.  Also one more question which is k8 version used in your setup

**Comment by AKamyshnikova** [comment]
I have v1.20 based k8s setup.

**Pull request merged**